### PR TITLE
Add Arab Bank Australia PDR

### PR DESCRIPTION
**Pull request by JayCheong**
Hi,

Please add Arab Bank Australia's PRD to the comparator hub.

Thanks,
Jay

### CODE REVIEW

File: public/datasources.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [
+    {"name": "Arab Bank Australia Limited", "url": "https://openbanking-api.arabbank.com.au/public/cds-au/v1", "icon": "https://www.arabbank.com.au/themes/arabbank/images/abal-logo.svg"}
```
Comment (vadkor) [inline public/datasources.json:2]: Missing `,` at the end of the closing brace.

**Review by vadkor — CHANGES_REQUESTED** [review]
Please fix the absent comma and your PR will be merged.

File: public/datasources.json (hunk 1 above)
Comment (JayCheong) [reply public/datasources.json]: Comma has been added, please review and approve if no issues.

Thanks,
Jay

**Review by vadkor — APPROVED** [review]

**Comment by nils-work** [comment]
Hi @vadkor 
The sort order for this entry has impacted the top 4, I'm not sure if that was intended.

**Comment by vadkor** [comment]
Thank you, @nils-work. The top 4 should have been left untouched. I'll fix that.

**Pull request merged**